README.md

```diff
@@ -1,28 +1,25 @@
-Chitter Challenge
-=================
-
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
-
-Challenge:
--------
-
-As usual please start by forking this repo.
-
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
-
-Features:
--------
+# Chitter Challenge - Week 4 Makers
+This is the weekend challenge for the Makers Academy for week 4. It is a basic clone of some of Twitter's functionality: The ability to sign up, log in, tweet, and log out.
+
+## To install
+- git clone "https://github.com/t-kellett/chitter-challenge"
+- run `bundle install` (assuming you have bundler installed, and a recent version of Ruby - this is built in Ruby 3.0.0)
+- run `psql`
+- run the sql commands in the subdirectory `db/migrations`
```

Comment:
Your setup instructions don't include steps on creating the databases, only creating the tables.